### ISSUE TITLE
It should stop reloading the page once user goes to payment page

### ISSUE THREAD

**Issue by GorvGoyl** [LeninGangwal/Big-Basket-Delivery-Slots#9]
Thanks  for your efforts! Was able to order within 5 minutes :)
Now, Once I see that slot is available and click on pay now button, your script still loads the screen :( due to which payment get stuck and need to pay later after bb notification that payment was unsuccessful.
Can we stop reloading the page if user goes to payment page?

**Comment by nileshjha19** [comment]
Yes. If we go by the bigbasket rules, we are anyway allowed only one active order at a time.
After the slots are found, you can break out of the while loop and run the application again for the next day. Something like this:

`if "checkout" in driver.current_url and not "Unfortunately, we do not have" in src:
                printTime("Found the slots!") 
                notify("Slots Available!","Please go and choose the slots")
                 break`
